ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -234,6 +235,12 @@ object Main {
     )
     private var experimental: Boolean = false
 
+    @Option(names = arrayOf("--ktlintignore"), description = arrayOf("Path to .ktlintignore (defaults to working dir)"))
```

Comment:
Would they pass in a path or would it happen by convention such as working directory plus home directory similar to .gitignore? I guess global prefs make less sense here -- possibly at each package level? Although I'm not sure if it's easy to support or even worth it.

Reply:
For `kotlinter-gradle` I suppose I'd start by saying the file had to be in the gradle project root. Of course ktlint CLI is agnostic to the idea of a project.

Reply:
If you look a bit further down, it does default to ".".

And yes, I was unsure whether we want/need to support multiple ignore/config files. I can certainly imagine a scenario (eg codegen files) but I'm unsure how common it would be. It seems like the biggest ask is for global disabling.

I originally considered a `.ktlintconfig` with the same syntax as `.editorconfig` so you could easily disable rules on a per-path basis but thought maybe the simpler approach was fine for now.

Reply:
I would agree with @shashachu, that it should be possible to add additionally rules to exclude in subprojects and `.ktlintconfig` should behave like `.editorconfig`.